src/silo/storage/column_group.cpp

```diff
@@ -66,10 +67,7 @@ void ColumnPartitionGroup::addValueToColumn(
          float_columns.at(column_name).insert(value.ToString());
          return;
    }
-   SPDLOG_ERROR(
-      "Unknown column type '{}' for column '{}'", static_cast<int>(column_type), column_name
-   );
-   abort();
+   UNREACHABLE();
```

Comment:
Fair enough, as in UNREACHABLE("because of ... reasoning"). I'll hold off, could do that in a separate PR. There's also the idea to make an ASSERT_EQ(a, b) which I also held back.

Reply:
See #559